cis194/week1/robert-chang/Homework01/test/Homework01Spec.hs

```diff
@@ -0,0 +1,42 @@
+module Homework01Spec where
+import Homework01 
+
+import Test.Hspec
+
+spec :: Spec
+spec = do
+  describe "toDigits" $ do
+   it "returns an empty list for a negative integer" $ do
+      toDigits (-1) `shouldBe` []
```

Comment:
Does it look better without the `()`?

Reply:
Let's think about this: Is `-1` a function or negative one?

Reply:
negative one